package.json

```diff
@@ -85,7 +90,7 @@
     "typescript": "^4.0.5"
   },
   "scripts": {
-    "build": "NODE_ENV=production CI=false npm-run-all -s build:css build:js",
+    "build": "NODE_ENV=production NODE_OPTIONS='--max-old-space-size=4096' CI=false npm-run-all -s build:css build:js",
```

Comment:
Hmm any reason for increasing the memory size?

Reply:
Yes, if we don't increase mem-size, the app will not be able to build with ci dude to the inclusion of opengsn... not really sure of the reason but ci shows error:
https://github.com/TradeTrust/document-creator-website/runs/1394465585?check_suite_focus=true